examples/aperture/README.rst

```diff
@@ -50,3 +50,54 @@ We run the following script to analyze correctness:
    .. literalinclude:: analysis_aperture.py
       :language: python3
       :caption: You can copy this file from ``examples/aperture/analysis_aperture.py``.
+
+
+.. _examples-aperture-pepperpot:
+
+Aperture Collimation with Periodic Masking
+===========================================
+
+Proton beam undergoing collimation by a periodic array of rectangular apertures, such as those used in a pepperpot emittance measurement.
+
+We use a 250 MeV proton beam with a horizontal rms beam size of 1.56 mm and a vertical rms beam size of 2.21 mm.
+
+After a short drift of 0.123 m, the beam intercepts a periodic array of apertures of period 1 mm (in the horizontal and vertical), each of which is 0.15 mm x 0.1 mm in size.
+
+In this test, the initial values of :math:`\sigma_x`, :math:`\sigma_y`, :math:`\sigma_t`, :math:`\epsilon_x`, :math:`\epsilon_y`, and :math:`\epsilon_t` must agree with nomin>
```

Comment:
Just double checking that this isn't a typo, what is "nomin>" here?

Reply:
Yes, that's a copy-paste error.  Thanks for catching it.